tests/test_requests.py

```diff
@@ -154,6 +154,22 @@ def test_HTTP_302_ALLOW_REDIRECT_GET(self, httpbin):
         assert r.history[0].status_code == 302
         assert r.history[0].is_redirect
 
+    def test_HTTP_307_ALLOW_REDIRECT_POST(self, httpbin):
+        parts = urlparse(httpbin('post'))
+        url = "HTTP://" + parts.netloc + parts.path
+        r = requests.post(httpbin('redirect-to'), data='test', params={'url': url, 'status_code': 307})
+        assert r.status_code == 200
+        assert r.history[0].status_code == 307
+        assert r.history[0].is_redirect
+
+    def test_HTTP_307_ALLOW_REDIRECT_POST_WITH_SEEKABLE(self, httpbin):
+        parts = urlparse(httpbin('post'))
+        url = "HTTP://" + parts.netloc + parts.path
```

Comment:
I thought to show a test where it works, and another where it doesn't, if it's redundant it can be removed.